appendices/VK_EXT_custom_border_color.txt

```diff
@@ -0,0 +1,89 @@
+include::{generated}/meta/VK_EXT_custom_border_color.txt[]
+
+*Last Modified Date*::
+    2019-10-14
+*IP Status*::
+    No known IP claims.
+*Contributors*::
+  - Joshua Ashton
+  - Liam Middlebrook, Nvidia
+
+This extension provides cross-vendor functionality to specify a custom border color
+for use when the sampler address mode ename:VK_SAMPLER_ADDRESS_MODE_CLAMP_TO_BORDER is used.
+
+To create a sampler which uses a custom border color
+slink:VkSamplerCreateInfo::pname:borderColor must be set to ename:VK_BORDER_COLOR_CUSTOM_EXT.
+
+When ename:VK_BORDER_COLOR_CUSTOM_EXT is supplied, applications must provide a
+slink:VkSamplerCustomBorderColorCreateInfoEXT in the pNext chain for
+slink:VkSamplerCreateInfo.
+
+For implementations with a limited number of border colors, they should be reference counted
+when the sampler is created/destroyed. This means that freeing the last sampler that used
+a custom border color will also free up the slot for another border color to be used.
+
+The implementation should maintain the guarantee that flink:vkCreateSampler and
```

Comment:
Should or must?  Should is frowned upon by the style guide since it implies optional behavior.  This isn't normative, but as Jeff mentions, best to stick to clear wording regardless.

Reply:
IMO this sentence can just be deleted.